wire-ios/Wire-iOS/Sources/UserInterface/Sketchpad/ColorKnobView.swift

```diff
@@ -23,6 +23,8 @@ final class ColorKnobView: UIView {
 
     // MARK: - Properties
 
+    let knobOuterDiameter: CGFloat = 18
```

Comment:
the value is taken from Figma https://www.figma.com/design/sOAbcd6UVtlZ5ejK4BoEvX/iOS?node-id=10718-127010&t=A8sbUOwJx8rpPFvX-0